server/aap/commands/fulfill_image_assignments.py

```diff
@@ -120,7 +124,8 @@ def _get_outstanding_photo_assignments(self):
         :return: The Id of any sheduled picure assignments
         """
         service = superdesk.get_resource_service('assignments')
-        assignments = service.find(where={"assigned_to.state": ASSIGNMENT_WORKFLOW_STATE.ASSIGNED,
+        assignments = service.find(where={"assigned_to.state": {"$in": [ASSIGNMENT_WORKFLOW_STATE.ASSIGNED,
```

Comment:
It would be better for this to use elasticsearch rather than mongodb

Reply:
Done